dpdk/vr_dpdk_knidev.c

```diff
@@ -324,6 +324,8 @@ vr_dpdk_knidev_init(struct vr_interface *vif)
         else
             /* ...so we use os_idx instead */
             port_id = vif->vif_os_idx;
+    } else if (vif->vif_type == VIF_TYPE_MONITORING) {
+            port_id = 0; /* we always use DPDK port 0 */
```

Comment:
Is this guaranteed to exist i.e. will rte_eth_dev_info_get() always return sane values for this port id?